index.js

```diff
@@ -64,15 +64,21 @@ function commist() {
     return args
   }
 
-  function register(command, func) {
+  function register(inputCommand, func) {
+    var command = inputCommand
+    var strict = false
+    if(typeof command === 'object'){
```

Comment:
Can you add a space between if and parentheses?

It seems this is missing linting (lol!)

Reply:
Added `standard` to the project